app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -76,6 +76,36 @@ def bulk_search
         render :json => ret
       end
 
+      api :PUT, '/tasks/bulk_resume', 'Resume all paused error tasks'
+      def bulk_resume
+        scope = resource_scope_for_index.select('DISTINCT foreman_tasks_tasks.*')
+        scope = scope.where(:state => :paused)
+        scope = scope.where(:result => :error)
```

Comment:
Ack on that: especially the search option accepting the scoped search can be really useful